src/components/Button/Button.plugin.js

```diff
@@ -0,0 +1,109 @@
+/* eslint-disable import/no-extraneous-dependencies */
+const _ = require('lodash')
+const chroma = require('chroma-js')
+const defaultConfig = require('tailwindcss/defaultConfig')()
+
+function defaultOptions() {
+  return {
+    borderRadius: '1.5rem',
+    fontWeight: '600',
+    lineHeight: '1.25',
+    fontSize: '1rem',
+    padding: '.5rem 1rem',
+    colors: {
+      white: {
+        background: defaultConfig.colors.white,
+        text: defaultConfig.colors.black,
+      },
+      black: {
+        background: defaultConfig.colors.black,
+        text: defaultConfig.colors.white,
+      },
+    },
+    sizes: {
+      small: {
+        fontSize: '.7rem',
+        padding: '.3rem .6rem',
+        borderRadius: '1.0rem',
+      },
+      large: {
+        fontSize: '1.25rem',
+        padding: '.75rem 1.5rem',
+        borderRadius: '1.7rem',
+      }
+    }
+  }
+}
+
+module.exports = (opts) => {
+  const options = _.isFunction(opts)
+   ? opts(defaultOptions())
+   : _.defaults(opts, defaultOptions())
+
+  return ({ addComponents, e }) => {
+    addComponents([
+      {
+        '.Button': {
```

Comment:
curious what your thoughts are on moving the `.Button` declarations to tailwind.js and then just pull it in here via `require`. Thinking behind that is the tailwind.js can serve as a quick theme reference for classes

Reply:
Then we would presumably want all class definitions in tailwind.js? The current layout seems to suggest that each component is fully contained within its folder, so referring back to tailwind.js seems counter-intuitive to me.

Reply:
good point